libraries/shared/storage/utils/decoder.go

```diff
@@ -43,6 +47,11 @@ func decodeUint256(raw []byte) string {
 	return n.String()
 }
 
+func decodeUint128(raw []byte) string {
```

Comment:
I was thinking about that, and then also was wondering if it would make sense for each of these decode methods to do some sort of validation that the bytes passed in are the right length?

Reply:
That seems like it would be cool, but do you know whether we can assume that input will be of a fixed length? Thinking that a `uint128` could be 16 bytes if it's packed, or 32 if it's not. I suppose we could validate that the input is greater than the min required?

Reply:
oh yeah, that's a great point. validating a minimum requirement seems doable, but i'm not sure how useful that may be.

for now, i'll update this to have just one `decodeInteger` method.